src/CommunityToolkit.Maui.MediaElement/Views/MauiMediaElement.windows.cs

```diff
@@ -9,9 +25,17 @@ namespace CommunityToolkit.Maui.Core.Views;
 /// </summary>
 public class MauiMediaElement : Grid, IDisposable
 {
+	bool navbarExist;
+	bool isDisposed;
+
+	static readonly AppWindow? appWindow = GetAppWindowForCurrentWindow();
 	readonly MediaPlayerElement mediaPlayerElement;
 
-	bool isDisposed;
+	static ImageSource source { get; set; } = new BitmapImage(new Uri("ms-appx:///fullscreen.png"));
```

Comment:
```suggestion
	static ImageSource source = new BitmapImage(new Uri("ms-appx:///fullscreen.png"));
```

Reply:
Thanks @pictos for the recommendation. It has been applied. Marked as resolved.